### PR TITLE
Avatars in chat

### PR DESCRIPTION
**Pull request by filipot**
I haven't gotten to try if this works, due to getting this error:
```
Error when sending contact request: TypeError: Cannot read property 'publicUri' of undefined
```
when clicking "last opp" when trying to upload a picture for my avatar

### CODE REVIEW

**Comment by askbk** [comment]
You don't have keys to the Cloud bucket :smile: If you need to test something related to avatars, you can manually insert some placeholders into the database (`profilePicturePath` in the `users` table), e.g. `https://api.adorable.io/avatars/285/abott@adorable.png`

File: frontend/src/components/Chat/index.jsx [hunk 1]
```diff
@@ -22,30 +22,33 @@ const Chat = ({ chatId }) => {
     const chatName = currConversation
         ? `${currConversation.members[0].firstName} ${currConversation.members[0].lastName}`
         : 'Samtale';
-    const [user, setUser] = useState({ firstName: '', lastName: '' });
-
     let messagebar = React.createRef();
 
+    const [user, setUser] = useState(null);
```
Comment (askbk) [inline frontend/src/components/Chat/index.jsx]: All members in the conversations can be accessed by `currConversation.members`, see line 23. No need to fetch users manually :)

Comment (filipot) [reply frontend/src/components/Chat/index.jsx]: I tried to print out currConversation and members only contained the other person, if not i'm mistaken.

**Comment by filipot** [comment]
2019-09-14-1568497558_screenshot_1920x1080


Chat seems to work as expected.

Except that there seems to be a problem for the first messages, as they are out of view at the top (you can't see them). Will look into it

File: frontend/src/components/Chat/index.jsx (hunk 1 above)
Comment (filipot) [reply frontend/src/components/Chat/index.jsx]: The currConversation only got one member, which is the other person. This is a print for currConversation, which contains the array member with a size of 1.
````
members: Array(1)
0:
birthDate: null
createdAt: "2019-09-06T10:55:44.000Z"
firstName: "filip"
id: 10
isAdmin: null
lastName: "heya"
municipality: null
phoneNumber: null
profileDescription: null
profilePicturePath: "https://api.adorable.io/avatars/285/abott@adorable.png"
updatedAt: "2019-09-06T10:55:44.000Z"
userType: "employee"
````

Comment (askbk) [reply frontend/src/components/Chat/index.jsx]: Should be pretty easy to include both people in the array :+1: can look into it later.

**Pull request merged**